pkg/registry/batch/job/storage/storage_test.go

```diff
@@ -132,12 +135,117 @@ func TestUpdate(t *testing.T) {
 	)
 }
 
-func TestDelete(t *testing.T) {
-	storage, server := newStorage(t)
-	defer server.Terminate(t)
-	defer storage.Job.Store.DestroyFunc()
-	test := genericregistrytest.New(t, storage.Job.Store)
-	test.TestDelete(validNewJob())
+func TestJobDeletion(t *testing.T) {
+	ctx := genericapirequest.NewDefaultContext()
```

Comment:
add a dummy warning recorder to the context (`WithWarningRecorder`) to intercept whether warnings were added